app/controllers/api/v1/packages_controller.rb

```diff
@@ -1,7 +1,6 @@
 module Api
```

Comment:
Great work !

**Out of scope for this PR:**

I do think this controller needs some extra refactoring

The problem is mostly due to the fact that both _updates_ and _creates_ use the same entry point. Which creates really convoluted code full of potential issues.
It also checks if an inventory number is present, and creates the PackagesInventory's "inventory" row when it detects that the inventory number has been set. We should probably have an `/packages/123/inventory` endpoint to make the whole thing more explicit (just thinking out loud)

Any volunteers ? 🤣 

@swatijadhav @abulsayyad123 @bharat619 @shreyas098 @steveyken

Reply:
Yep, let's do it. Good to add a ticket for this and pick in the next sprint.